pages/hub/introduction/usecases.md

```diff
@@ -4,27 +4,27 @@ import WithMDX from '../../../lib/with-mdx'
 import { nelson } from '../../../lib/data/team'
 
 export const page = {
-title: 'Hub Usecases',
+title: 'Hub Use cases',
```

Comment:
Could you lowercase the `use` here please?

Reply:
Fixed